spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -57,8 +57,11 @@ def get_parser():
     seg = parser.add_argument_group('\nTASKS')
     seg.add_argument(
         "-task",
+        nargs="+",
         help="Task to perform. It could either be a pre-installed task, task that could be installed, or a custom task."
-             " To list available tasks, run: sct_deepseg -list-tasks",
+             " To list available tasks, run: sct_deepseg -list-tasks. To use a custom task, indicate the path to the "
+             " ivadomed packaged model. More than one path can be indicated for cascaded application of the models. "
```

Comment:
hum... I feel we need to create a specific section on the ivadomed RTD that explains what we mean by "packaged model". It is not clear that we mean a folder, including a .pt and .json having the same basename as the folder name. 

In terms of order, in order to save one PR: we could first clarify on ivadomed, and then add the URL in this function usage.

Reply:
> hum... I feel we need to create a specific section on the ivadomed RTD that explains what we mean by "packaged model". It is not clear that we mean a folder, including a .pt and .json having the same basename as the folder name.

We could move some of t[hese info](https://github.com/ivadomed/ivadomed/wiki/repositories#models) from the wiki to the RTD, eg [here](https://ivadomed.org/en/latest/pretrained_models.html#pre-trained-models). + add info about json and .onnx etc.

Reply:
Yes I think we could add this information in [Pre-trained models ](https://ivadomed.org/en/latest/pretrained_models.html#pre-trained-models). I think it is fine to leave the conventions where they are since it is more for ivadomed developers.

Reply:
yep we agree about where to move info